lib/widget/map.dart

```diff
@@ -103,12 +106,19 @@ class _MapState extends State<Map> {
           zoomGesturesEnabled: true,
           polygons: buildingHighlights,
           polylines: direction,
+          markers: markers,
           initialCameraPosition: _cameraPosition,
           onMapCreated: (GoogleMapController controller) {
             _completer.complete(controller);
           },
           onCameraMove: (CameraPosition cameraPosition) async {
             GoogleMapController _mapController = await _completer.future;
+            //removing markers if the zoom is greater than 20
```

Comment:
Should this require a setState(){}?

Reply:
it works fine without setState(), would you please run it on your device to double check?

Reply:
yea it works without setState